---
title: "Microbial community profiling for human microbiome projects: \n Tools, techniques, and challenges"
subtitle: "Michael Hamady & Rob Knight. Genome Res. 2009."
author: "Presented by Kelly Sovacool"
date: "23 Sept. 2019"
output:
  xaringan::moon_reader:
    lib_dir: libs/
    chakra: libs/remark-latest.min.js
    css: libs/remark-css-0.0.1/default.css
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, eval = TRUE)
```
```{r render, eval=FALSE}
library(xaringan)
rmarkdown::render("microbiome-profiling.Rmd", output_dir = "docs/")
```

## Outline

- Background
- Key Questions
- Techniques in microbial community profiling
- Discussion
- My Conclusions

---

### Background

**Microbiome**: The collection of microbes that live in an environment.

.center[
```{r, out.width= '80%'}
knitr::include_graphics("figures/Microbiome_spheres_2016_EMBL.jpg")
```
]

.footnote[
[1] [EMBL-EBI: What's a microbiome](https://www.ebi.ac.uk/about/news/feature-story/whats-a-microbiome)
]

---

### Background: Why care?

Changes in the composition of the human microbiome are associated with health and disease.

.center[
```{r, out.width= '80%'}
knitr::include_graphics("figures/human_microbiome.gif")
```
]

.footnote[
[2] [The Human Microbiome Project, Nature 2012](https://dx.doi.org/10.1038%2Fnature11234) [3] [theconversation.com/the-human-microbiome](http://theconversation.com/the-human-microbiome-is-a-treasure-trove-waiting-to-be-unlocked-118757)
]

---
### Background
- **Microbial community profiling**: Characterize **which microbes are present** and how **abundant** they are.

---
### Background
- **Microbial community profiling**: Characterize **which microbes are present** and how **abundant** they are.

- Application: detetecting colorectal cancer
.center[
```{r, out.width='100%'}
knitr::include_graphics("figures/baxter_2016_genome-med.png")
```
]

.footnote[
[4] [Baxter et al., Genome Med. 2016](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-016-0290-3)
]

---

### Background: Operational Taxonomic Unit (OTU)

- Proxy for microbial species at different taxonomic levels.
- Created by clustering sequences based on sequence similarity.
- Avoids difficulties of classifying bacteria into species when unknown organisms are present.

```{r, out.width='100%'}
knitr::include_graphics("figures/OTU_clustering.png")
```

---

### Key Questions Raised

- Is there a core set of microbial species or genes that all humans share?

- How do changes in microbial abundances at particular taxonomic levels relate to health & disease?

---
## Techniques: Overall Workflow

.center[
```{r, out.width='100%'}
knitr::include_graphics("figures/F2.large.jpg")
```
]
Fig. 2
---
### Techniques: Sequencing

#### Metagenomics _vs._ 16S rRNA

- Metagenomics
    - Whole-genome shotgun sequencing.
    - Measures functional potential of the whole microbiome.
    - Need longer reads and greater sampling depth to discover rare sequences.
    
- 16S rRNA
    - Only sequence part of the 16S rRNA gene, which is highly conserved across microbial species.
    - Useful for comparing at different taxon levels (phylum through species) becauase some regions evolve quickly while others evolve more slowly.
    - Sufficient to estimate which microbes are present at the genus level and their abundances.
    - Long reads and high sampling depth aren't necessary.

---

### Techniques: Sequencing

- Selecting regions of the 16S rRNA gene. 
- V2 and V4 yield similar results to the full-length gene. 
- V6 is not recommended due to many unclassified sequences.

---

### Techniques: Clustering OTUs

.pull-left[
```{r, out.width='60%'}
knitr::include_graphics("figures/multi-omics_16S-rRNA.jpg")
```
]

.pull-right[
- Cluster similar 16S sequences into Operational Taxonomic Units (OTUs).

- Compare abundances of OTUs within and across samples.
]

.footnote[
[5] [Bikel et al., Comput. Struct. Biotech. 2015](10.1016/j.csbj.2015.06.001)
]

---

### Techniques: OTU clustering algorithms

- Nearest-neighbor: greedily add a sequence if it is similar to *any* sequence in the OTU
- Furthest-neighbor: only add a sequence if it is similar to *all* other sequences in the OTU

.center[
```{r, out.width='100%'}
knitr::include_graphics("figures/F5.E-F.jpg")
```
]
Fig. 5 E & F

---

### Techniques: Comparing Communities

.pull-left[Phylum-level] .pull-right[OTU-level (proxy for species)]


.center[
```{r, out.width='100%'}
knitr::include_graphics("figures/F3.large.jpg")
```
]
Fig. 3: 22 human gut samples show quite different abundances even at the phylum level .

---

## Discussion: Key Questions Addressed

- Is there a core set of microbial species or genes that all humans share?
  - Microbiomes between individuals vary greatly by which species are present and their abundances.
  - No species are shared at high abundance.
  
- How do changes in microbial abundances at particular taxonomic levels relate to health & disease?
  - Focus on gene sets, rather than taxon levels, to compare functional potential of microbiomes.

---

## My Conclusions

- Discussion of tree-based vs taxon-based methods concluded that tree-based methods are better, but didn't provide substantial evidence.
- Lacked substantial discussion of the strenghs & weakenesses of different clustering algorithms.
  - How does the quality of OTU assignments produced by different algorithms compare.

---

## Advancements

- mothur & QIIME
- optiClust

---

## Towards integrated omics

https://www-sciencedirect-com.proxy.lib.umich.edu/science/article/pii/S2001037015000318#f0005


